user-interface/7001-COLL-collateral.md

```diff
@@ -1 +1,18 @@
-# Collateral
\ No newline at end of file
+# Collateral
+
+## Collateral Data Grid
+
+- **Must** be shown grouped by asset (<a name="7001-COLL-001" href="#7001-COLL-001">7001-COLL-001</a>)
+- For each asset:
+  - **Must** show used amount, which is the total of all Margin accounts (<a name="7001-COLL-002" href="#7001-COLL-002">7001-COLL-002</a>)
+  - **Must** show available amount, which is the total of your General account (<a name="7001-COLL-003" href="#7001-COLL-003">7001-COLL-003</a>)
+  - **Must** show total amount, which is the sum of your General account and all Margin accounts (<a name="7001-COLL-004" href="#7001-COLL-004">7001-COLL-004</a>)
+  - **Must** provide a method for depositing asset (<a name="7001-COLL-005" href="#7001-COLL-005">7001-COLL-005</a>)
+  - **Must** provide a method for withdrawing asset (<a name="7001-COLL-006" href="#7001-COLL-006">7001-COLL-006</a>)
+  - **Must** show the asset symbol (<a name="7001-COLL-007" href="#7001-COLL-007">7001-COLL-007</a>)
```

Comment:
Request (non-blocking): I think the data appendix could handle the display of symbol VS Name (similar to market name/code). 
Generally name is required as assets can share a symbol, but the name should help make users disambiguate. 
however, if its a list you look at lots, in a small space, the symbol alone might be enough, as long as you can click something and get the name and contract address.